tests/Makefile

```
@@ -19,79 +16,108 @@ all: minikube operator test
 # Start minikube
 .PHONY: minikube
 minikube:
-	minikube start --driver=docker
-	helm repo update
+	@echo "===== Starting minikube cluster. ====="
+	@minikube start --driver=docker
 
 # Install official k8s-agents-operator repo
 .PHONY: operator
 operator:
-	helm uninstall k8s-agents-operator --ignore-not-found
-	helm repo add k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
-	helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
+	@echo "===== Removing any existing operator deployments. ====="
+	@helm uninstall k8s-agents-operator --ignore-not-found
+	@sleep 1
+	@echo "===== Adding and updating operator helm chart. ====="
+	@helm repo add --force-update k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
+	@echo "===== Deploying operator to minikube cluster. ====="
+	@helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
 		--set=licenseKey=${NEW_RELIC_LICENSE_KEY} \
 		--set=controllerManager.manager.image.version=edge
-	sleep 5
-	kubectl wait --for=condition=Ready pods $$(kubectl get pods | grep k8s-agents-operator | cut -d" " -f1)
+	@sleep 5
+	@kubectl wait --for=condition=Ready pods $$(kubectl get pods | grep k8s-agents-operator | cut -d" " -f1)
+	@echo "===== Giving operator time to generate certificates. ====="
+	@sleep 30
```

This has a chance to not be enough time, and that could cause downstream commands to fail I assume. What about this if we know the cert names?
```suggestion
	@kubectl wait --for condition=established --timeout=60s crd/<crd-name-here>
```

---

If we encounter issues in the future with the timing I try that. For now it seems to be working so we're going to unblock all the queued releases